src/schema/provenance_schema.yaml

```diff
@@ -874,6 +880,11 @@ ENTITIES:
     excluded_properties_from_public_response:
       - lab_tissue_sample_id
       - submission_id
```

Comment:
Add `metadata.lab_id`, for when the sample itself contains such info nested under `Sample.metadata`

Reply:
done